quantlib/handle.pxd

```diff
@@ -16,11 +16,12 @@ cdef extern from 'ql/handle.hpp' namespace 'QuantLib':
         Handle(T*)
```

Comment:
This is unrelated to this PR, but I think this constructor has been deprecated since QuantLib 1.4 (https://github.com/lballabio/quantlib/blob/QuantLib-v1.4/QuantLib/ql/handle.hpp#L80)